charts/k8ssandra-cluster/templates/stargate/stargate.yaml

```diff
@@ -46,12 +46,11 @@ spec:
             - name: CLUSTER_VERSION
               value: {{ .Values.stargate.clusterVersion | quote }}
             - name: SEED
-              value: {{ .Values.clusterName + "-seed-service" | quote }}
```

Comment:
Helm didn't like this syntax. I'm not sure why.

Reply:
The templating engine does not have a string concatenation operator. You don't need it though. It can be written as,

```yaml
{{ .Values.clusterName }}-seed-service"
```

The only env vars that need to use the `quote` function are `CLUSTER_VERSION` and `ENABLE_AUTH` assuming it is templated as well.

Reply:
I thought it was Go templating, and that as such I could use any valid Go expression. Is that not the case?